shared/components/NextCommunityEvent/index.js

```diff
@@ -46,30 +49,22 @@ export function getHeaderText(startDateTime, endDateTime) {
   }
 }
 
-const ReadMoreButton = ({ onClick }) => (
-  <a href="#" className="NextCommunityEvent__readmore" onClick={onClick}>
-    Read more
-  </a>
-);
-
-ReadMoreButton.propTypes = {
-  onClick: PropTypes.func,
-};
-
 class NextCommunityEvent extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      collapsed: true,
       statusProps: getTicketStatusOptions(props),
     };
+  }
 
-    this.expandInfo = this.expandInfo.bind(this);
+  componentDidUpdate(preProps) {
```

Comment:
I wouldn't do this using componentDidUpdate. Instead of (assuming this is old React) I would use a render prop, 
 this is the best example I could find of the pattern I had in mind (https://codedaily.io/tutorials/6/Using-Functions-as-Children-and-Render-Props-in-React-Components). But essentially in the render you would have something like `<EventStatus {...props}>{(statusProps)=>{...`.